lib/fictium/exporters/api_blueprint_exporter/action_formatter.rb

```diff
@@ -0,0 +1,29 @@
+module Fictium
+  class ApiBlueprintExporter
+    class ActionFormatter
+      def format(action)
+        [build_header(action).strip, build_examples(action)].select(&:present?).join("\n\n")
```

Comment:
:eyes: You often are using `select(&:present?).join("\n\n")`, you can delegate it in BaseExporter